server/templates/widget.js

```diff
@@ -84,7 +84,7 @@
     this.loading.className = 'oc-loading-container';
     this.logo = document.createElement('img');
     this.logo.className = 'oc-loading';
-    this.logo.src = '{{host}}/static/images/opencollective-icon.svg';
+    this.logo.src = '{{host}}/static/images/opencollective-icon.png';
```

Comment:
Did you try this to make sure it looks well? This logo is rotated from the center, we must make sure the new one is acting properly.

Gif preview welcome :-)

Reply:
Yes I don't see any problems with this. 😉 

![widgets](https://user-images.githubusercontent.com/12435965/167507048-f644bd03-99a5-47fb-bf2a-dc3a01f351ba.gif)